src/topology/subset_properties.lean

```diff
@@ -627,6 +627,18 @@ iff.intro (assume h, h.image hf.continuous) $ assume h, begin
   rwa [hf.induced, nhds_induced, ←map_le_iff_le_comap]
 end
 
+/-- A closed embedding is proper, i.e., inverse images of compact sets are compact. -/
+lemma closed_embedding.tendsto_cocompact
+  {f : α → β} (hf : closed_embedding f) : tendsto f (filter.cocompact α) (filter.cocompact β) :=
+begin
+  rw filter.has_basis_cocompact.tendsto_iff filter.has_basis_cocompact,
+  intros K hK,
+  refine ⟨f ⁻¹' (K ∩ (set.range f)), _, λ x hx, by simpa using hx⟩,
+  apply hf.to_embedding.compact_iff_compact_image.mpr,
+  convert hK.inter_right hf.closed_range,
```

Comment:
`convert` seems a bit strange here. Can't you simply `rw  set.image_preimage_eq_of_subset (set.inter_subset_right _ _)`  first and then `exact hK.inter_right hf.closed_range`?